api/v1alpha1/hosted_controlplane.go

```diff
@@ -37,6 +37,9 @@ type HostedControlPlaneSpec struct {
 	Platform     PlatformSpec                `json:"platform"`
 	DNS          DNSSpec                     `json:"dns"`
 
+	// +optional
+	FIPS bool `json:"fips"`
```

Comment:
in future, we may want to fail creation of this cluster's control plane if its not rendered to a fips enabled mgmt cluster.